controllers/ibmvpccluster_controller.go

```diff
@@ -71,6 +71,11 @@ func (r *IBMVPCClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, err
 	}
 
+	// Determine whether the Cluster is designed for extended Infrastructure support, implemented in a separate path.
+	if ibmCluster.Spec.Network != nil {
```

Comment:
this code block can above line 64 to avoid one extra Get call..

Reply:
I think this needs to be here, after the `ibmCluster` variable is defined and then populated via `r.Get` from the Cluster resource on line 65-66.

Reply:
yep, my bad..